components/brave_new_tab_ui/storage/new_tab_storage.ts

```diff
@@ -28,6 +28,7 @@ export const defaultState: NewTab.State = {
   customLinksNum: 0,
   showRewards: false,
   showBraveTalk: false,
+  showBraveVPN: false,
```

Comment:
do we still need this? I think it can be removed (and it's never set to anything else)

Reply:
It's updated whenever prefs for showing the card is changed.
```
    case types.NEW_TAB_PREFERENCES_UPDATED:
      const preferences = payload as NewTab.Preferences
      const newState = {
        ...state,
        ...preferences
      }
     ....
```